### PR TITLE
create a param to force recalculation of formula fields when data is …

### PR DESCRIPTION
**Pull request by jamesduncan**
…updated.

### CODE REVIEW

File: dataFields/ABFieldFormulaCore.js [hunk 1]
```diff
@@ -81,7 +81,14 @@ module.exports = class ABFieldFormulaCore extends ABField {
       delete values[this.columnName];
    }
 
-   format(rowData) {
+   /**
+    * @method format
+    * both calculate and format the data input based of user settings
+    * for this field.
+    * @param {obj} rowData a key=>value hash of the current values.
```
Comment (Hiro-Nakamura) [inline dataFields/ABFieldFormulaCore.js:88]: In my comment headers, I've been trying to change our formatting to be like so:
```
* @param {type} {param}
*        {start the description here, but keep the lines to <= 80 cols }
*        {use as many lines as you need to }
```

It keeps to the ESLINT rule of <= 80 cols and actually gives us a bit more space for writing the comments.

**Review by Hiro-Nakamura — APPROVED** [review]
From what I can tell, this all looks fine here.

File: dataFields/ABFieldFormulaCore.js (hunk 1 above)
Comment (jamesduncan) [reply dataFields/ABFieldFormulaCore.js]: I made the change to the formatting.

File: dataFields/ABFieldFormulaCore.js [hunk 2]
```diff
@@ -81,7 +81,16 @@ module.exports = class ABFieldFormulaCore extends ABField {
...
 
-   format(rowData) {
+   /**
+    * @method format
+    * both calculate and format the data input based of user settings
+    * for this field.
+    * @param {obj} rowData
+    *        a key=>value hash of the current values.
+    * @param {boolean} reCalculate
+    *        a boolean that signals if we should force recalculation of values
```
Comment (jamesduncan) [inline dataFields/ABFieldFormulaCore.js:91]: How does this look?

**Pull request merged**